README.md

```diff
@@ -129,6 +129,7 @@ Documentation:	https://github.com/StyraInc/regal/blob/main/docs/rules/style/use-
 Regal comes with a set of built-in rules, grouped by category.
 
 - **bugs**: Common mistakes, potential bugs and inefficiencies in Rego policies.
+- **custom**: Custom, or organizational rules, where you decide what to enforce.
```

Comment:
```suggestion
- **custom**: Custom, rules where enforcement can be adjusted to match your organization or team preferences.
```

Reply:
Thanks! I shortened it down to just "your preferences" so that we wouldn't have a linke break in that list.